projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk-jupyterlab-extension/tests/test_handlers.py

```diff
@@ -3,13 +3,98 @@
 import json
 
 
-async def test_get_example(jp_fetch):
+async def test_run_job_get(jp_fetch):
     # When
-    response = await jp_fetch("vdk-jupyterlab-extension", "get_example")
+    response = await jp_fetch("vdk-jupyterlab-extension", "run", method="GET")
+
+    # Then
+    assert response.code == 200
+
+
+async def test_run_job_post(jp_fetch):
+    # When
+    body = {"jobPath": "", "jobArguments": ""}
+    response = await jp_fetch(
+        "vdk-jupyterlab-extension", "run", method="POST", body=json.dumps(body)
+    )
+
+    # Then
+    assert response.code == 200
+    payload = json.loads(response.body)
+    assert payload == {"message": "0"}
+
+
+async def test_delete_job_post(jp_fetch):
+    # When
+    body = {"jobName": "", "jobTeam": "", "restApiUrl": ""}
+    response = await jp_fetch(
+        "vdk-jupyterlab-extension", "delete", method="POST", body=json.dumps(body)
+    )
+
+    # Then
+    assert response.code == 200
+    payload = json.loads(response.body)
+    assert payload == {
+        "error": "true",
+        "message": "¯\\_(ツ)_/¯\n"
+        "\n"
+        "what: Control Service Error\n"
+        "why: No host specified.\n"
+        "consequences: Operation cannot complete.\n"
+        "countermeasures: Verify that the provided url (--rest-api-url) is "
+        "valid and points to the correct host.\n",
+    }
+
+
+async def test_download_job_post(jp_fetch):
```

Comment:
test rest api url empty ?

Reply:
I am testing only the handlers, not the operations. So I would say it doesn't matter whether it is a wrong input or not. It just returns an error message, but is still 200